controllers/user.js

```diff
@@ -1,3 +1,4 @@
+import bcryptjs from "bcryptjs";
 import { PrismaClient } from "@prisma/client";
 
```

Comment:
Since money has been moved to the saveGame objects,
The getUsers and getUser methods still have `select: { money: true }`
which makes both methods show a prisma error

You may want to use the prisma function `omit`
This way each time the schema for user changes, the prisma query shouldnt have to change
```js 
const user = await prisma.user.findUnique({
  "where": { "id": String(req.params.id) },
  "omit": {
    password: true
  },
});
```
Note it is a preview feature and I have not researched if this means it may come with issues
https://www.prisma.io/docs/orm/prisma-client/queries/excluding-fields

Reply:
Didn't know this was a thing! Thanks for pointing that out. If it is a preview feature it may be included after the current version of `PrismaClient` we are running. There is another way to fix this that I will use if this doesn't work!

Reply:
Unfortunately it isn't part of the version we're using and don't want to accidentally create more bugs. For now I'm just going to remove `money` from the requests instead. The fix I mentioned works for all single user queries and will use that there.

Reply:
it is unfortunate that the version of prisma client we are running doesnt have the `omit` feature,
but the same end result has be implemented through clean js instead